homeassistant/components/doorbird.py

```diff
@@ -151,28 +172,203 @@ def custom_url(self):
         """Get custom url for device."""
         return self._custom_url
 
-    @property
-    def monitored_events(self):
-        """Get monitored events."""
-        if self._monitored_events is None:
-            return []
+    def update_schedule(self, hass):
+        """Register monitored sensors and deregister others."""
+        from doorbirdpy import DoorBirdScheduleEntrySchedule
+
+        # Create a new schedule (24/7)
+        schedule = DoorBirdScheduleEntrySchedule()
+        schedule.add_weekday(0, 604800)  # seconds in a week
+
+        # Get the URL of this server
+        hass_url = hass.config.api.base_url
+
+        # Override url if another is specified in the configuration
+        if self.custom_url is not None:
+            hass_url = self.custom_url
+
+        # For all sensor types (enabled + disabled)
+        for sensor_type in SENSOR_TYPES:
+            name = '{} {}'.format(self.name, SENSOR_TYPES[sensor_type]['name'])
+            slug = slugify(name)
+
+            url = '{}{}/{}?token={}'.format(hass_url, API_URL, slug,
```

Comment:
I know that this is an old PR but this doesn't make sense anymore. We now have a webhook component that does exactly this.

Reply:
Do you have a link to some documentation or maybe some other components that have implemented via webhook component?

Reply:
IFTTT component.